client/cmd.cpp

```diff
@@ -35,11 +37,12 @@ void cmd_help(char*) {
 
 	printf(
 		"\n"
-		"You can also use the TAB key to autocomplete commands\n"
+		"See man pbc(1) for more info about specific commands\n"
+		"Hint: you can use the TAB key to autocomplete commands\n"
 	);
 }
 
-void cmd_send(char* addr_str) {
+void cmd_send(char * addr_str) {
```

Comment:
```suggestion
void cmd_send(char* addr_str) {
```
Stay consistent!1!!!1!11!1!